code/modules/ordeals/steel_ordeals.dm

```diff
@@ -1,26 +1,28 @@
 /datum/ordeal/simplespawn/steel_dawn
 	name = "Dawn of Steel"
 	annonce_text = "The augmentation... I don’t regret it. One downside, though, is that I won’t ever be able to return to civilian life."
-	annonce_sound = 'sound/effects/ordeals/indigo_start.ogg'
-	end_sound = 'sound/effects/ordeals/indigo_end.ogg'
+	annonce_sound = 'sound/effects/ordeals/indigo_start.ogg' // Give it proper sounds
```

Comment:
New sounds for the ordeal are made by coxwane. I have them downloded now but i can place them here.